src/cli-client/internals/cli/cli_upload.go

```diff
@@ -0,0 +1,108 @@
+package cli
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/canonical/oci-factory/cli-client/internals/client"
+	"github.com/canonical/oci-factory/cli-client/internals/logger"
+	"github.com/canonical/oci-factory/cli-client/internals/trigger"
+)
+
+type UploadRelease struct {
+	Track     string
+	Risks     []string
+	EndOfLife string
+}
+
+type CmdUpload struct {
+	UploadRelease []string `long:"release" description:"Release images to container registries.\nSyntax: --release tracks=<release track>,risks=<risk1>[,<risk2>...],eol=yyyy-mm-dd"`
+}
+
+var cmdUpload CmdUpload
```

Comment:
Why is this a global variable? I think you can achieve the same by doing:
```go
parser.Command.AddCommand("upload", "Trigger the build and release for the image in the current working directory", "long description", &CmdUpload{})
```

Reply:
Good catch. Fixed.